docs/action-debugging.md

```diff
@@ -0,0 +1,26 @@
+# Debugging
+If the build logs do not provide enough detail on why a build may be failing, some other options exist to assist with troubleshooting.
+
+## Runner Diagnostic Logs
+Runner Diagnostic Logs provide insight on how the Runner is executing an action. This information is provided in two files, the runner file and the worker file.
+
+These files contain the prefix `Runner_` or `Worker_` to indicate the file type.
+
+Each log file contains different logging information that corresponds to that process:
+  * The Runner process coordinates setting up workers to execute jobs.
```

Comment:
I didn't want to go too in depth here, is this appropriate?

Reply:
Yeah, this seems fine to me